components/dashboard/src/settings/SelectIDE.tsx

```diff
@@ -176,20 +177,45 @@ function orderedIdeOptions(ideOptions: IDEOptions) {
         });
 }
 
-function renderIdeOption(option: IDEOption, selected: boolean, onSelect: () => void): JSX.Element {
+function renderIdeOption(
+    option: IDEOption,
+    selected: boolean,
+    version: IDEOption["imageVersion"],
```

Comment:
**question**: When selecting the latest release, VS Code version remains the same. Is this expected?

Reply:
I think here it's because in preview environments, we just use the stable version as Insiders. Should be expected

@iQQBot please correct me if I'm wrong here 😄 🙏

Reply:
Thanks, @filiptronicek! OK, let's see how this behaves in production, then. 🎣